src/js/tech/tech.js

```diff
@@ -437,6 +437,15 @@ class Tech extends Component {
     return '';
   }
 
+  static isTech(component) {
+    /* jshint proto:true */
+    let proto = component.__proto__;
+    /* jshint proto:false */
+    if (Object.getPrototypeOf) {
```

Comment:
Looks IE10 has `Object.getPrototypeOf` but it doesn't give us the correct answer.